src/components/cookie-message/style.less

```diff
@@ -1,30 +1,58 @@
 :host {
-  background-color: #1d1d1d;
-  color: #FFF;
-  font-family: "Scania Sans Semi Condensed";
-  font-size: 1.7rem;
 
-  padding: 26px 30px 26px 30px;
-  height: auto;
-  width: 100%;
+  .cookie-short-message {
```

Comment:
This isn't used anywhere?

Reply:
Everything that is inside .cookie-short-message  should instead be inside host, or the template needs a change